webapp/components/test-search.html

```diff
@@ -170,7 +156,10 @@
         const value = e.target.value.toLowerCase();
 
         if (opts.includes(value)) {
-          this.commitQuery();
+          this.dispatchEvent(new CustomEvent('autocomplete', {
```

Comment:
As discussed offline, we may want to drop this and let the `'commit'` listener detect when a query matches a path.

Reply:
I think we still want to expose the `autocomplete` event. I doubt a user will type a full path (though they may paste one), and we don't want to need a double-enter for autocompletion.